chainstate/test-suite/src/tests/fungible_tokens.rs

```diff
@@ -380,6 +381,71 @@ fn token_issue_test(#[case] seed: Seed) {
     });
 }
 
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn token_issue_skip_on_transferred_amount_check(#[case] seed: Seed) {
```

Comment:
Please add some docs explaining what this tests... having trouble understanding.

Reply:
the test is actually not needed at all